ansible/vars/all.yml

```diff
@@ -10,6 +10,8 @@ vagrant_local:
 php:
     install: '1'
     ppa: php
-    packages: [php7.0-cli, php7.0-intl, php7.0-mcrypt, php7.0-mysql, php7.0-curl, php7.0-json, php7.0-xml]
+    packages: [php7.0-cli, php7.0-intl, php7.0-mcrypt, php7.0-mysql, php7.0-curl, php7.0-json, php7.0-xml, php7.0-mbstring]
+    pecl_packages: [php-zip]
```

Comment:
After adding php-mbstring `composer install` fails with this message
```
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
    Failed to download psr/http-message from dist: The zip extension and unzip command are both missing, skipping.
The php.ini used by your command-line PHP is: /etc/php/7.0/cli/php.ini
    Now trying to download from source
  - Installing psr/http-message (1.0.1)
    Cloning f6561bf28d520154e4b0ec72be95418abe6d9363

                                                                                                                                    
  [RuntimeException]                                                                                                                
  Failed to clone https://github.com/php-fig/http-message.git, git was not found, check that it is installed and in your PATH env.  
  sh: 1: git: not found
```
So php-zip is also required